src/core.py

```diff
@@ -11,6 +11,8 @@
 from src.utils.image import CLAHE_HELPER, ImageUtils
 from src.utils.interaction import InteractionUtils
 
+auto_align_constant_220 = 220
+auto_align_constant_200 = 200
```

Comment:
these constants need to be semantically named. So that in case we change their value next time, we don't need to rename the constant.

Also they should be placed inside the main constants file or config.json file.